extensions/kafka/pom.xml

```diff
@@ -110,6 +110,17 @@
             <version>${testcontainers.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.avro</groupId>
+            <artifactId>avro</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-schema-registry</artifactId>
+            <version>6.2.2</version>
```

Comment:
Maybe move `confluent.version` from `hazelcast-sql/pom.xml` to the root pom and reuse it here?